src/ipfsd-daemon.js

```diff
@@ -239,7 +239,24 @@ class Daemon {
       return this
     }
 
-    await this.api.stop()
+    let killTimeout
+    let killed = false
+    if (this.opts.forceKill !== false) {
```

Comment:
can you add default to true here https://github.com/ipfs/js-ipfsd-ctl/blob/3dae27670fdebe0edd6e1e26ddf5df4c274b62bf/src/factory.js#L20 and in the docs